packages/queries/src/validators/validateNestedQueriesHaveAValidRefOp.ts

```diff
@@ -0,0 +1,23 @@
+import { QueryBuilderError } from '../QueryBuilderError';
+import { Query } from '../types/types';
+import { isRefOp } from '../types/isRefOp';
+
+/**
+ Validate that every included sub-query has at least one RefOp
+ */
+export function validateNestedQueriesHaveAValidRefOp<DB>(query: Query<DB>) {
+    const nestedQueries = Object.entries(query.include ?? {});
```

Comment:
just realised: you're using `entries` but only actually using the values. Same below. Just use `Object.values`.

More attention to details please :)